js/esbuild.js

```diff
@@ -0,0 +1,72 @@
+const argv = require('yargs').argv;
+const { watch } = argv;
+const esbuild = require('esbuild');
+
+const target = 'es2020';
+const outdir = 'dist';
+const sourcedir = 'src';
+// This message must match the script ready regexp in packages\devops\tasks\develop.js
+const successMessage = '[esbuild] scripts build succeeded';
```

Comment:
I don't think outdir, sourcedir, successMessage are needed in this version

Reply:
updated